src/common/config/constants.ts

```diff
@@ -285,21 +285,14 @@ export const OPERATOR_NAME_CNV = 'cnv';
 export const OPERATOR_NAME_LSO = 'lso';
 export const OPERATOR_NAME_ODF = 'odf';
 export const OPERATOR_NAME_LVM = 'lvm';
-export const OPERATOR_NAME_CVO = 'cvo';
-export const OPERATOR_NAME_CONSOLE = 'console';
```

Comment:
I removed the ones (with the corresponding translation stuff) that were unused in the codebase.

Reply:
We should check with CIM as I know there are projects that have references to `assisted-ui-lib`